### DISCUSSION TITLE
Deployer locally in project and globally

### DISCUSSION THREAD

**Discussion by jasperf** [deployphp/deployer#2732] [Help needed]
When I use Deployer now in my project it seems to load the installed package inside the project, but when I check for package used it shows the global package:

```
jasper@MacMini  ~/code/domain.com/domain.com   master  dep -v --version
Using /Users/jasper/code/domain.com/domain.com/vendor/bin/dep
Deployer dev-master
jasper@MacMini  ~/code/domain.com/domain.com   master  which dep
/Users/jasper/.composer/vendor/bin/dep
```

Any ideas why this is the case? I know it may be a *Nix issue really, but just wondering. 

Also, do you normally recommend a local or global installation?

**Comment by antonmedv** [comment]
This is a feature of Deployer. If you have a global installation of dep and run it in the folder with local installed Deployer, global dep will redirect call to local dep.

I recommend installing it globally (for `dep init` command to be available, autocomplete) and locally for project-specific requirements.

**Reply by jasperf** [reply]
I see. Did not know about that redirect. That is a good solution and good to know Deployer takes care of this redirect to locally installed Deployer version if available.
